src/shared/components/InputBase/InputBase.styles.ts

```diff
@@ -1,101 +1,67 @@
 import { colors, sizes, transitions, typography } from '@/shared/theme'
-import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
 type FormGroupProps = {
   error?: boolean
   disabled?: boolean
 }
 
-export const FormGroup = styled.div<FormGroupProps>`
-  position: relative;
+export const FormGroup = styled.label<FormGroupProps>`
+  display: inline-block;
+  width: 100%;
   cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'auto')};
+  opacity: ${({ disabled }) => (disabled ? 0.5 : 1)};
+
+  input:-webkit-autofill,
+  input:-webkit-autofill:hover,
+  input:-webkit-autofill:focus,
+  textarea:-webkit-autofill,
+  textarea:-webkit-autofill:hover,
+  textarea:-webkit-autofill:focus {
+    // This is madness, but basically, it will make autofill transparent for a long time.
+    // Changing background-color and color won't work
+    transition: color 99999s, background-color 99999s;
+    transition-delay: 99999s;
+  }
+  input:-webkit-autofill::first-line,
+  textarea:-webkit-autofill::first-line {
+    font-size: 1rem;
+  }
+
   input,
-  button {
+  button,
+  textarea {
     transition: all ${transitions.timings.regular} ${transitions.easing};
```

Comment:
Because of this, resizing the textarea feels lagged. Let's explicitly specify properties that should transition here